Manager.php

```diff
@@ -247,7 +247,7 @@ private function pathify($format)
         if (is_array($format)) {
             return md5(serialize($format));
         } else {
-            return $format;
+            return str_replace(array('<', '>'), array('L','G'), $format);
```

Comment:
Be sure all special characters are handled, not only the one you use now.

Check https://github.com/snowcap/SnowcapImBundle/blob/master/docs/extra.rst to see example of other characters used.